doc/FAQ.org

```diff
@@ -264,12 +264,16 @@ binding in the mode's map, e.g. for =magit-status-mode=,
 #+END_SRC
 
 ** Include underscores in word motions?
-You can modify the syntax table of the mode in question. For example, for Python
-mode:
+You can modify the syntax table of the mode in question. To do so you can
+include this on your =dotspacemacs/user-init=.
```

Comment:
You're both right. I just tried it again  and it worked on `dotspacemacs/user-config`. I'll fix the doc. Thanks!